<?xml version="1.0" encoding="UTF-8"?>
<record_update table="sys_ui_message">
    <sys_ui_message action="INSERT_OR_UPDATE">
        <application>Attestation</application>
        <key>attestationInvalideUser</key>
        <language>en</language>
        <message>The attestation for "{0}" failed because the response email was not sent by the owner.&#13;
The owner is {1} with email address {2}.</message>
        <sys_class_name>sys_ui_message</sys_class_name>
        <sys_created_by>rbeeman</sys_created_by>
        <sys_created_on>2018-05-04 23:13:06</sys_created_on>
        <sys_customer_update>false</sys_customer_update>
        <sys_id>d183a6ba13f51bc01c7851522244b0b4</sys_id>
        <sys_mod_count>1</sys_mod_count>
        <sys_name>attestationInvalideUser</sys_name>
        <sys_package display_value="Attestation" source="x_thimi_attest">420892240ff1dbc062b6e64be1050e71</sys_package>
        <sys_policy/>
        <sys_replace_on_upgrade>false</sys_replace_on_upgrade>
        <sys_scope display_value="Attestation">420892240ff1dbc062b6e64be1050e71</sys_scope>
        <sys_update_name>sys_ui_message_d183a6ba13f51bc01c7851522244b0b4</sys_update_name>
        <sys_updated_by>rbeeman</sys_updated_by>
        <sys_updated_on>2018-05-04 23:13:50</sys_updated_on>
    </sys_ui_message>
</record_update>
